SignalMessaging/utils/ConversationStyle.swift

```diff
@@ -137,7 +137,7 @@ public class ConversationStyle: NSObject {
     private static let defaultBubbleColorIncoming = UIColor.ows_messageBubbleLightGray
 
     @objc
-    public let bubbleColorOutgoingUnsent = UIColor.ows_light10
+    public let bubbleColorOutgoingFailed = UIColor.ows_darkSkyBlue
```

Comment:
Huh, so it's now the same as the "sent" color?

Reply:
According to zeplin.